source/_integrations/tag.markdown

```diff
@@ -93,6 +93,62 @@ automation:
 To find your scanner's device ID, open Developer tools -> Events -> Listen to events and subscribe to `tag_scanned`.
 Then scan a tag on the reader and note down the `device_id` from the `data` section. 
 
+Another example is using tags for disarming the home assistant alarm system. 
+
+{% raw %}
+automation:
+  - id: TAG.001
+    alias: "Handle Tag Scan"
```

Comment:
While a cool example, it is IMHO, extremely over-complicated to serve as an example.
Also, is it not formatted according to our documentation and YAML style guidelines, please make sure it fits those.